src/Microsoft.AspNet.Http/RequestCookies.cs

```diff
@@ -0,0 +1,467 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNet.Http.Internal
+{
+    public sealed class RequestCookies : IRequestCookies
```

Comment:
This statement confused me; then I had a look at request cookies and it really isn't a collection is it! Will change :)